src/System.Private.CoreLib/src/System/Utf8String.Construction.cs

```diff
@@ -49,7 +50,7 @@ private Utf8String Ctor(ReadOnlySpan<byte> value)
 
             Utf8String newString = FastAllocate(value.Length);
             Buffer.Memmove(ref newString.DangerousGetMutableReference(), ref MemoryMarshal.GetReference(value), (uint)value.Length);
-            return Utf8Utility.ValidateAndFixupUtf8String(newString);
+            return Utf8Utility.ValidateAndFixupUtf8String(newString)!;
```

Comment:
TODO-COMMENT? This can only return null if input is null, right?

Reply:
Yup.